umake/frameworks/games.py

```diff
@@ -210,3 +210,42 @@ def post_install(self):
                         exec='"{}" %f'.format(self.exec_path),
                         comment=self.description,
                         categories="Development;IDE;"))
+
+
+class Superpowers(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, category):
+        super().__init__(name="Superpowers", description=_("The HTML5 2D+3D game maker"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         download_page="https://github.com/superpowers/superpowers/releases/latest",
```

Comment:
This is the page that is downloaded, isn't it? It seems that your medium assets are refering to a tagged version (tests/data/server-content/github.com/superpowers/superpowers/releases/tag/v0.19.0/index.html).

I may have missed something, but are you sure that in the medium tests you are thus testing the mock asset? :)
(if it's supposed to be a redirect, there is one example in one of the framework to test this)